integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -855,7 +857,9 @@ private void callWebAppAndCheckForServerNameInResponse(
     // map with server names and boolean values
     HashMap<String, Boolean> managedServers = new HashMap<String, Boolean>();
     for (int i = 1; i <= TestUtils.getClusterReplicas(domainUid, clusterName, domainNS); i++) {
-      managedServers.put(domainUid + "-" + managedServerNameBase + i, new Boolean(false));
+      String msnb =
+          managedServerNameBase.toLowerCase(Locale.ENGLISH).replace('_', '-').replace('.', '-');
+      managedServers.put(domainUid + "-" + msnb + i, new Boolean(false));
```

Comment:
I think you can replace the characters '_ 'and '.' with '-' for adminServerName and managedServerNameBase  in the map itself in initialize() method after loading the yaml to map.